test/functional_test.go

```diff
@@ -284,21 +279,8 @@ func AssertExpectations(
 			assert.FailNow(t, err.Error())
 		}
 
-		// Fake manifest
-		m, err := manifest.NewManifest(api.ProjectId(), api.Host(), testhelper.NewMemoryFs())
-		if err != nil {
-			assert.FailNow(t, err.Error())
-		}
-
 		// Load actual state
-		schedulerApi, _, _ := testapi.NewMockedSchedulerApi()
-		logger, _ := utils.NewDebugLogger()
-		stateOptions := state.NewOptions(m, api, schedulerApi, context.Background(), logger)
-		stateOptions.LoadRemoteState = true
-		actualState, ok := state.LoadState(stateOptions)
-		assert.True(t, ok)
-		assert.Empty(t, actualState.RemoteErrors().Errors)
-		actualSnapshot, err := state.NewProjectSnapshot(actualState, project)
+		actualSnapshot, err := project.NewSnapshot()
```

Comment:
Je to teraz cele jednoduchsie nemusi sa robit `fake manifest`, ani sa nemoze nastat problem, ze sa nejaka komponenta v `State` ignoruje, ako sa stalo tebe.